1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts

```diff
@@ -1,4 +1,31 @@
+import { palindromeChecker } from './index'
 
 describe('palindrome checker', () => {
 
+    it('should exist', () => {
+        expect(palindromeChecker).toBeDefined();
+    });
+
+    it ('returns true for single word palindrome examples - mom, Mom, MoM, xMomx', () => {
```

Comment:
You could use parameterization to avoid adding examples in the test name (spec)
example:
```ts
describe("Can recognize palindromes of small letters words", () => {
  it.each([
    ["mom", true],
    ["wow", true],
    ["bill", false]
  ])("knows that '%s' is a palindrome", (word, expected) => {
    expect(isPalindrome(word)).toBe(expected);
  });
});

describe("Can recognize palindromes even when the input consists of multiple words", () => {
  it.each([
    ["Was It A Rat I Saw", true],
    ["Never Odd or Even", true],
    ["Never Odd or Even1", false]
  ])("knows that '%s' is a palindrome", (phrase, expected) => {
    expect(isPalindrome(phrase)).toBe(expected);
  });
});
```